core/app/helpers/spree/products_helper.rb

```diff
@@ -30,11 +30,12 @@ def variant_full_price(variant)
 
     # converts line breaks in product description into <p> tags (for html display purposes)
     def product_description(product)
-      if Spree::Config[:show_raw_product_description]
-        raw(product.description)
+      description = if Spree::Config[:show_raw_product_description]
+        product.description
```

Comment:
This seems to be the right way of doing it else I will have to assign attribute in each if-else clause

```
  if Spree::Config[:show_raw_product_description]
    description = product.description
  else
    description = product.description.to_s.gsub(/(.*?)\r?\n\r?\n/m, '<p>\1</p>')
  end
```